quiche/src/h3/mod.rs

```diff
@@ -1670,6 +1793,25 @@ impl Connection {
 
         conn.stream_send(stream_id, grease_payload, false)?;
 
+        trace!(
+            "{} tx frm GREASE stream={} len=18",
+            conn.trace_id(),
+            stream_id
+        );
+
+        qlog_with_type!(QLOG_FRAME_CREATED, conn.qlog, q, {
+            let frame = Http3Frame::Reserved { length: Some(18) };
+            let ev_data = EventData::H3FrameCreated(H3FrameCreated {
+                stream_id,
+                length: Some(18),
+                frame,
+                raw: None,
+            });
+
+            q.add_event_data_with_instant(ev_data, std::time::Instant::now())
```

Comment:
Why not just use `add_event()`?

Reply:
TL;DR is that every `event` is unique, with unique `event data` and a timestamp. `add_event()` requires the app to include more boilerplate code. Other qlog streamer functions provide some syntactic sugar to avoid that. 

What we are missing is a `add_event_data_now()` method that would shorten this line. Adding that wouldn't be too hard if you like.

Reply:
Ok I pushed a new commit that adds add_event_data_now to the qlog crate (and fixes the rustdoc to make it clearer how these functions differ. Updated the H3 change commit to use that and save some lines,